.devcontainer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/ubcsailbot/sailbot_workspace/dev:fix-building-pre-base
+FROM ghcr.io/ubcsailbot/sailbot_workspace/dev:combined-pip3-installs3
```

Comment:
Have you updated this to the latest tag? If so, good to merge

Reply:
Will update now